code/game/objects/effects/anomalies/anomalies_ectoplasm.dm

```diff
@@ -0,0 +1,205 @@
+				else if(prob(10))
+					new /obj/effect/decal/cleanable/dirt/dust(get_turf(impacted_thing))
+
+				if(!isplatingturf(impacted_thing))
+					var/turf/open/floor/floor_to_break = impacted_thing
+					if(floor_to_break.overfloor_placed && floor_to_break.floor_tile && prob(20))
+						new floor_to_break.floor_tile(floor_to_break)
+						floor_to_break.make_plating(TRUE)
+						floor_to_break.broken = TRUE
+						floor_to_break.burnt = TRUE
+
+			if(ishuman(impacted_thing))
+				var/mob/living/carbon/human/mob_to_infect = impacted_thing
+				mob_to_infect.ForceContractDisease(new /datum/disease/revblight(), FALSE, TRUE)
+				new /obj/effect/temp_visual/revenant(get_turf(mob_to_infect))
+				to_chat(mob_to_infect, span_revenminor("A cacophony of ghostly wailing floods your ears for a moment. The noise subsides, but a distant whispering continues echoing inside of your head..."))
+
+			if(istype(impacted_thing, /obj/structure/window))
+				var/obj/structure/window/window_to_damage = impacted_thing
+				window_to_damage.take_damage(rand(60, 90))
+				if(window_to_damage?.fulltile)
+					new /obj/effect/temp_visual/revenant/cracks(get_turf(window_to_damage))
+
+	if(effect_power >= 35)
+		var/effect_range = ghosts_orbiting + 3
+		haunt_outburst(get_turf(src), effect_range, 45, 2 MINUTES)
+
+	if(effect_power >= 50) //Summon a ghost swarm!
+		var/list/candidate_list = list()
+		for(var/mob/dead/observer/orbiter in orbiters?.orbiter_list)
+			candidate_list += orbiter
+
+		new /obj/structure/ghost_portal(get_turf(src), candidate_list)
+
+		priority_announce("Anomaly has reached critical mass. Ectoplasmic outburst detected.", "Anomaly Alert")
+
+/**
+ * Manages updating the sprite for the anomaly based on how many orbiters it has.
+ *
+ *
+ * A check that is run to determine which sprite the anoamly should currently be displaying.
+ * With 50% or more participation, the "heavy" sprite is used. Otherwise, it is reverted to the normal anomaly sprite.
+ */
+
+/obj/effect/anomaly/ectoplasm/proc/intensity_update()
+	if(effect_power >= 50) //If we're at the threshold for the highest tier effect, we change sprites in preparation for the spooks.
+		icon_state = "ectoplasm_heavy"
+		update_icon_state()
+	else
+		icon_state = "ectoplasm"
+		update_icon_state()
+
+
+
+// Ghost Portal. Used to bring anomaly orbiters into the playing field as ghosts. Destroys itself and all of its associated ghosts after two minutes.
+// Can be destroyed early to the same effect.
+
+/obj/structure/ghost_portal
+	name = "Spooky Portal"
+	desc = "A portal between our dimension and who-knows-where? It's emitting an absolutely ungodly wailing sound."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "anom"
+	anchored = TRUE
+	var/static/list/spooky_noises = list('sound/hallucinations/growl1.ogg','sound/hallucinations/growl2.ogg','sound/hallucinations/growl3.ogg','sound/hallucinations/veryfar_noise.ogg','sound/hallucinations/wail.ogg')
```

Comment:
```suggestion
	var/static/list/spooky_noises = list(	
		'sound/hallucinations/growl1.ogg',
		'sound/hallucinations/growl2.ogg',
		'sound/hallucinations/growl3.ogg',
		'sound/hallucinations/veryfar_noise.ogg',
		'sound/hallucinations/wail.ogg',
	)
```

Reply:
actually don't commit this suggestion to your branch i don't trust that weird tab github is adding there